apcd-cms/src/apps/submission_form/urls.py

```diff
@@ -1,7 +1,7 @@
-from django.urls import re_path
+from django.urls import path
 from .views import SubmissionFormView
 
 app_name = 'apcd'
 urlpatterns = [
-    re_path('', SubmissionFormView.as_view(), name='index')
+    path('request-to-submit/', SubmissionFormView.as_view(), name='index')
```

Comment:
**Thoughts.** I like this, because it's easier to know where the URLs are set (I forget to go outside of the `apps/` dir and into `urls_custom.py`). But please see [my `urls_custom.py` comment](...).